components/applications-service/pkg/storage/postgres/hab_event.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/chef/automate/api/external/applications"
 	"github.com/chef/automate/api/external/habitat"
+	"github.com/chef/automate/components/applications-service/pkg/storage"
 	"github.com/go-gorp/gorp"
```

Comment:
[nit] It's nice if the imports are sorted; first stdlib, then third-party deps, then internal deps. `goimports -local github.com/chef/automate` supports that.

Reply:
I am supposedly running that command in my vim editor, not sure why it is not sorting them! 🤔

Reply:
Is this how you want them¿?
```go
import (
	"fmt"
	"strings"
	"time"

	dblib "github.com/chef/automate/lib/db"

	"github.com/go-gorp/gorp"
	"github.com/golang/protobuf/ptypes"
	timestamp "github.com/golang/protobuf/ptypes/timestamp"
	"github.com/pkg/errors"
	"github.com/prometheus/client_golang/prometheus"
	"github.com/prometheus/client_golang/prometheus/promauto"
	log "github.com/sirupsen/logrus"

	"github.com/chef/automate/api/external/applications"
	"github.com/chef/automate/api/external/habitat"
	"github.com/chef/automate/components/applications-service/pkg/storage"
)
```

Reply:
💭  I'd cluster the dblib with the rest in the last section, but it's also a bit of a nitpicky comment anyways -- :+1: